docs/3.api/1.composables/use-id.md

```diff
@@ -0,0 +1,25 @@
+---
+title: "useId"
+description: The useId composable creates a unique id.
+---
+
+# `useId`
```

Comment:
Might be worth naming this something like `getUniqueID` rather than `use*` as the latter is more often associated with composables, but this doesn't rely on Nuxt or Vue context.

Reply:
maybe it's better to use a `uniqueID`/ `uniqueId`?

Reply:
`getUniqueID()`